src/gui/res/png2indexed.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+mkdir palette
+
+
+PNG_DIR=./png
+PAL_DIR=./palette
+
+png2indexed(){
+
+read input
+
+for image in $input 
+do
+	echo "$PNG_DIR/$image.png to $PAL_DIR/$image.png"
+	#uses image magick tool to convert png from normal to indexed 
+	magick "$PNG_DIR/$image.png" -type palette "$PAL_DIR/$image.png"
```

Comment:
introducing another tool (dependency) is not needed IMO when optipng can reduce bit depth as well:

```
optipng -fix -o7 -zc6 -zw32k -strip all arrow_down.png
** Processing: arrow_down.png                                                                                                                                                                                                                 
12x12 pixels, 4x8 bits/pixel, RGB+alpha                                                                                                                                                                                                       
Reducing image to 4 bits/pixel, 8 colors (7 transparent) in palette
....
```